src/common-utils/main.sh

```diff
@@ -311,6 +311,53 @@ install_alpine_packages() {
     PACKAGES_ALREADY_INSTALLED="true"
 }
 
+# Arch Linux packages
```

Comment:
Can we updates NOTES.md's  `OS support` section to include `Arch Linux`? 

https://github.com/devcontainers/features/blob/fff9a95f2242a79c83504bd9d4a56ad4b2fa2868/src/common-utils/NOTES.md#os-support